server/conn_test.go

```diff
@@ -1555,6 +1555,13 @@ func TestAuthTokenPlugin(t *testing.T) {
 	err = cc.openSessionAndDoAuth(resp.Auth, resp.AuthPlugin)
 	require.NoError(t, err)
 
+	// login succeeds even if the password expires now
```

Comment:
Could we rename this function from `TestAuthTokenPlugin` to `TestAuthSessionTokenPlugin`, since there is another plugin named `tidb_auth_token`.